Libraries/LibWeb/Painting/DisplayListRecorder.cpp

```diff
@@ -194,26 +194,57 @@ void DisplayListRecorder::draw_painting_surface(Gfx::IntRect const& dst_rect, No
     });
 }
 
+[[nodiscard]]
+static bool is_rect_correction_necessary(Gfx::ExifOrientation exif_orientation)
```

Comment:
Nit: This feels like it belongs in `ImageOrientation`, as something like `exif_orientation_affects_image_size()`.